pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -0,0 +1,141 @@
+package totalaccountwatcher
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/openshift/aws-account-operator/pkg/awsclient"
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/go-logr/logr"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	"github.com/openshift/aws-account-operator/pkg/localmetrics"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+const (
+	AwsSecretName = "aws-account-operator-credentials"
+)
+
+// ErrAwsAccountLimitExceeded indicates the orgnization account limit has been reached.
+var ErrAwsAccountLimitExceeded = errors.New("AccountLimitExceeded")
+
+// TotalAccountWatcher global var for TotalAccountWatcher
+var TotalAccountWatcher *totalAccountWatcher
+
+var log = logf.Log.WithName("aws-account-operator")
+
+type totalAccountWatcher struct {
+	watchInterval time.Duration
+	AwsClient     awsclient.Client
+	client        client.Client
+	Total         int
+}
+
+// Initialize creates a global instance of the TotalAccountWatcher
+func Initialize(client client.Client, watchInterval time.Duration) {
+	log.Info("Initializing the totalAccountWatcher")
+
+	AwsClient, err := awsclient.GetAWSClient(client, awsclient.NewAwsClientInput{
+		SecretName: AwsSecretName,
```

Comment:
I feel like this is already defined somewhere and can be used just like namespace, as an exported value

Reply:
It's exported in the account-controller. But importing that would create an import cycle.

Reply:
Can we move those values to utils or something?

Reply:
Done